metaflow/flowspec.py

```diff
@@ -288,40 +289,68 @@ def merge_artifacts(self, inputs, exclude=[]):
         inputs : List[Steps]
             Incoming steps to the join point
         exclude : List[str], optional
+            If specified, do not consider merging artifacts with a name in `exclude`.
+            Cannot specify if include is also specified
```

Comment:
Could you add `include` and `exclude` (using tilde) in L:293 and L:295?

Reply:
Done.